src/commands.py

```diff
@@ -463,6 +441,74 @@ async def reply(mes: discord.Message, _):
         else:
             await mes.channel.send(f"ERROR: While attempting to DM, there was an unexpected error. Tell aquova this: {err}")
 
+
+class GetUserForReplyException(Exception):
+    """
+    Helper exception to make _get_user_for_reply easier to write.
+
+    By raising this the function can bail out early, so fewer levels of if/else nesting are needed.
+    """
+    pass
+
+
+"""
+_get_user_for_reply
+
+Gets the user to reply to for a reply command.
+
+Based on the reply command staff wrote, and the channel it was sent in, this figures out who to DM.
+
+Returns a user (or None, if staff mentioned a user not in the server) and the number of words to strip from the reply command.
```

Comment:
> the number of words to strip from the reply command.

I know that I'm the one who set it up this way, but looking at it again, I wish there was a way to do this better; it seems very flimsy. The other thing I could think of is returning a formatted string from this function, but then it's serving quite a few purposes. May have to think on this.

Reply:
Yeah, I agree.

I can't think of an easy way without changing a bunch of code so I'd rather do that outside of this PR.

Reply:
That's fair, we can deal with that later.